src/Core/Framework/Adapter/Twig/SwTwigFunction.php

```diff
@@ -23,24 +27,25 @@ class SwTwigFunction
      *
      * @param mixed $object The object or array from where to get the item
      * @param mixed $item The item to get from the array or object
-     * @param array<mixed> $arguments An array of arguments to pass if the item is an object method
+     * @param array<int|mixed> $arguments An array of arguments to pass if the item is an object method
      * @param string $type The type of attribute (@see \Twig\Template constants)
      * @param bool $isDefinedTest Whether this is only a defined check
      * @param bool $ignoreStrictCheck Whether to ignore the strict attribute check or not
      * @param int $lineno The template line where the attribute was called
-     * @param bool $sandboxed When true the extension use SandboxExtension
+     *
+     * @throws RuntimeError if the attribute does not exist and Twig is running in strict mode and $isDefinedTest is false
      *
      * @return mixed The attribute value, or a Boolean when $isDefinedTest is true, or null when the attribute is not set and $ignoreStrictCheck is true
      *
      * @internal
      */
-    public static function getAttribute(Environment $env, Source $source, mixed $object, mixed $item, array $arguments = [], $type = /* Template::ANY_CALL */ 'any', $isDefinedTest = false, $ignoreStrictCheck = false, $sandboxed = false, int $lineno = -1)
+    public static function getAttribute(Environment $env, Source $source, mixed $object, mixed $item, array $arguments = [], $type = /* Template::ANY_CALL */ 'any', $isDefinedTest = false, $ignoreStrictCheck = false, bool $sandboxed = false, int $lineno = -1)
     {
         try {
+            FieldVisibility::$isInTwigRenderingContext = true;
             if ($object instanceof Struct) {
-                FieldVisibility::$isInTwigRenderingContext = true;
-
-                if ($type === Template::METHOD_CALL) { // @phpstan-ignore-next-line
+                if ($type === Template::METHOD_CALL) {
+                    // @phpstan-ignore-next-line
```

Comment:
please use `@phpstan-ignore` and the identifier of the issue, with a proper comment, why this could not be fixed. Also see https://phpstan.org/user-guide/ignoring-errors